UI/src/UserInterface.cpp

```diff
@@ -40,6 +61,7 @@ void UserInterface::run(
         {
             if (not m_paused)
             {
+                ScopedOn outputPause(m_paused);
```

Comment:
Actually, another thread can pause after the `if` read but before this! Race condition :)